quic/s2n-quic/src/tests.rs

```diff
@@ -10,11 +11,16 @@ use crate::{
     },
     Server,
 };
-use std::time::Duration;
+use std::net::SocketAddr;
+use std::{
+    sync::{Arc, Mutex},
+    time::Duration,
+};
 
 mod setup;
 use bytes::Bytes;
-use s2n_quic_platform::io::testing::primary;
+use s2n_quic_platform::io::testing::network::Packet;
+use s2n_quic_platform::io::testing::{primary, TxRecorder};
```

Comment:
This is missing a rustfmt, which is why the job is failing.

Reply:
I was a bit confused because everything should format on save, but looks like vscode picked up the stable toolchain from our `rust-toolchain`, but the rustfmt we are running in CI depends on nightly.
`rustup run nightly rustfmt quic/s2n-quic/src/tests.rs` <- fixed it.

Reply:
Actually, this is better.
```
rustup run nightly cargo fmt --all
```

Reply:
Unfortunately, the import formatting is nightly-only so it does require a nightly fmt. I think you do get warnings of it using unstable formatting configuration on stable, though.